packages/synapse-sdk/src/telemetry/adapters/sentry.ts

```diff
@@ -0,0 +1,170 @@
+import type { CustomEvent, HTTPEvent, OperationEvent, OperationType, TelemetryConfig } from '../types.ts'
+import { BaseTelemetryAdapter } from './base-adapter.ts'
+import { integrations, Sentry } from './sentry-dep.ts'
+
+/**
+ * Sentry telemetry adapter
+ * Works in both Node.js and browser - sentry-dep.js is swapped via package.json "browser" field
+ */
+export class SentryAdapter extends BaseTelemetryAdapter {
+  init(config: TelemetryConfig, tags: Record<string, string>): void {
+    Sentry.init({
+      dsn: 'https://3ed2ca5ff7067e58362dca65bcabd69c@o4510235322023936.ingest.us.sentry.io/4510235328184320',
+      // Setting this option to false will prevent the SDK from sending default PII data to Sentry.
+      // For example, automatic IP address collection on events
+      sendDefaultPii: false,
+      environment: config.environment || 'production',
+      beforeSend: this.sanitizeEvent,
+      // Enable tracing/performance monitoring
+      tracesSampleRate: 1.0, // Capture 100% of transactions for development (adjust in production)
+      // Integrations configured per-runtime in sentry-dep files
+      integrations,
+    })
+
+    Sentry.setContext('environment', {
+      sdkVersion: tags.sdkVersion,
+      runtime: tags.runtime,
+      network: tags.network,
+      ua: tags.ua,
+      appName: tags.appName,
+    })
+
+    // Set global tags
+    Sentry.setTag('sdk_version', tags.sdkVersion)
+    Sentry.setTag('runtime', tags.runtime)
+    Sentry.setTag('network', tags.network)
+
+    if (tags.appName) {
+      Sentry.setTag('app_name', tags.appName)
+    }
+  }
+
+  captureError(error: Error, context?: Record<string, unknown>): void {
+    Sentry.captureException(error, {
+      tags: context?.operation ? { operation: String(context.operation) } : undefined,
+      extra: context,
+    })
+  }
+
+  captureHTTP(event: HTTPEvent): void {
+    const startTime = new Date(event.ts).getTime()
+
+    const span = Sentry.startInactiveSpan({
+      name: `${event.method} ${event.urlTemplate}`,
+      op: event.spOperation,
+      startTime: startTime,
+      attributes: {
+        // Use standard searchable properties
```

Comment:
Can we link to the docs that defines these standard properties?  It looks like https://docs.sentry.io/concepts/search/searchable-properties/spans/ ?

Given tags are also searchable, I'm wondering the value of trying to conform to their standard properties vs. us just using our own set of tags that are maybe more self documenting?

Reply:
Tags were not searchable for me when i was testing. Maybe i missed something